Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests
+{
+  [TestFixture]
+  public class ChromeBinariesProviderTest
+  {
+    [Test]
+    public void GetInstalledExecutable ()
+    {
+      var provider = new ChromeBinariesProvider();
+
+      var result = provider.GetInstalledExecutable();
+
+      Assert.That (result, Is.Not.Null);

```

Comment:
I think this assert is unnecessary, as the test will fail regardless if `result` is null

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/12/commits/cc21cb58b42c8d277dc5010ce41de8afefffb1e3.